docs/resources/tuning_config.md

```diff
@@ -19,7 +19,7 @@ Edit a cluster tuning config
 
 - `cluster` (String) Identifier of the cluster. After the creation of the resource, it is not possible to update the attribute value.
 - `name` (String) Name of the tuning configuration. After the creation of the resource, it is not possible to update the attribute value.
-- `spec` (String) Definition of the spec
+- `spec` (String) Definition of the spec. It is required to supply this field wrapped in a jsonencode call. Example: jsonencode({<tuning_config_spec})
```

Comment:
I guess we should create a follow-up issue in TF improvements to investigate this. wdyt ?

Reply:
+1 lets capture a ticket for this and we can follow up with a spike to see if there is any other options